src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -66,6 +66,8 @@ struct alts_tsi_handshaker {
   bool shutdown = false;
   // Maximum frame size used by frame protector.
   size_t max_frame_size;
+  // The list of preferred transport protocols.
```

Comment:
Please also explain here that this string is a comma-separated list of transport protocols, sorted in order of preference.